cmd/admin-handlers.go

```diff
@@ -284,6 +284,7 @@ type ServerHTTPAPIStats struct {
 // including their average execution time.
 type ServerHTTPStats struct {
 	S3RequestsInQueue      int32              `json:"s3RequestsInQueue"`
+	ReachedS3Requests      uint64             `json:"reachedS3Requests"`
```

Comment:
This should be called incoming @vadmeste and has to be changed in this manner elsewhere